### DISCUSSION TITLE
Changing Default Icon

### DISCUSSION THREAD

**Discussion by sinarahmany** [NativePHP/laravel#234] [Ideas]
Hi, I`m trying to change the default icon but no luck so far. Any suggestions would be greatly appreciated!

**Comment by mattsplat** [comment]
I haven't found any built in way to do this but you can replace the icon file in the vendor folder vendor/nativephp/electron/resources/js/resources/icon.png

**Reply by sinarahmany** [reply]
Thanks, yeah, it's not the most convenient method, but it seems to be the only option for now.

**Comment by nakajisan** [comment]
As a workaround, you can change the default icon by adding the following to the project's composer.json, then running `composer install`.

```json
    "scripts": {
        "post-install-cmd": [
            "cp ./storage/app/icon.png ./vendor/nativephp/electron/resources/js/resources/icon.png"
        ],
    },
```

**Comment by simonhamp** [comment]
Yep, this isn't supported just yet. Moving to Ideas. [I did start on this](https://github.com/NativePHP/electron/pull/72) a while ago, but haven't finalised it yet

**Comment by simonhamp** [comment]
A quick update, I opened [a new PR for this](https://github.com/NativePHP/electron/pull/103) which is a smoother approach. Closing this thread as this will be merged soon.
